### PR TITLE
[Rom] Adapted compressible potential app to ROM usage

### PR DESCRIPTION
**Pull request by EduardGomezEscandell**
**📝 Description**
This PR does the bare minimum to allow running a compressible potential case in ROM.
Tested with the NACA0012 example in GiD problemtype examples.

Depends on #9662.

**🆕 Changelog**
- Created a solver wrapper in the `CompressiblePotentialFlowApplication`
- Added wrapper to `new_python_solvers_wrapper_rom.py`

### CODE REVIEW

**Review by rubenzorrilla — COMMENTED** [review]
Makes sense for me but the `CreateSolver` method in `PotentialFlowAnalysis` must be modified accordingly to use the new solvers wrapper (see the `CreateSolver` in `FluidDynamicsAnalysis` or `StructuralMechanicsAnalysis`.

Aside of this changes make sense for me.

**Comment by marcnunezc** [comment]
> Makes sense for me but the `CreateSolver` method in `PotentialFlowAnalysis` must be modified accordingly to use the new solvers wrapper (see the `CreateSolver` in `FluidDynamicsAnalysis` or `StructuralMechanicsAnalysis`.
> 
> Aside of this changes make sense for me.

I agree, I would say this can be done in this PR.

**Comment by EduardGomezEscandell** [comment]
Yep, makes sense to me.

@rubenzorrilla @marcnunezc

**Review by rubenzorrilla — COMMENTED** [review]
Makes sense for me. I let @marcnunezc to give the final approval.

**Comment by marcnunezc** [comment]
@EduardGomezEscandell  I think you undid your last changes with the last merge. Was that intentional?

**Comment by EduardGomezEscandell** [comment]
> @EduardGomezEscandell I think you undid your last changes with the last merge. Was that intentional?

Definitely not 😅.

Fixed

File: applications/CompressiblePotentialFlowApplication/python_scripts/potential_flow_analysis.py [hunk 1]
```diff
@@ -1,21 +1,8 @@
...
-            return potential_flow_solver.CreateSolver(self.model, self.project_parameters["solver_settings"])
-        elif self.project_parameters["solver_settings"]["solver_type"].GetString()=="ale_potential_flow":
-            import KratosMultiphysics.CompressiblePotentialFlowApplication.ale_potential_flow_solver as ale_potential_flow_solver
-            return ale_potential_flow_solver.CreateSolver(self.model, self.project_parameters["solver_settings"], self.parallel_type)
-        elif self.project_parameters["solver_settings"]["solver_type"].GetString()=="adjoint_potential_flow":
-            import KratosMultiphysics.CompressiblePotentialFlowApplication.potential_flow_adjoint_solver as adjoint_solver
-            return adjoint_solver.CreateSolver(self.model, self.project_parameters["solver_settings"])
-        else:
-            raise Exception("Solver type '"+str(self.project_parameters["solver_settings"]["solver_type"].GetString())+"' not added. Please specify an available solver")
+        return CreateSolverByParameters(self.model, self.project_parameters["solver_settings"], self.parallel_type)
```
Comment (marcnunezc) [inline applications/CompressiblePotentialFlowApplication/python_scripts/potential_flow_analysis.py]: I just checked the fluid analysis and there the function CreateSolver(model, params) is used instead, which then calls the CreateSolverByParams() function:

https://github.com/KratosMultiphysics/Kratos/blob/3badd462572ba10c8c8189d7f23df034a14762eb/applications/FluidDynamicsApplication/python_scripts/python_solvers_wrapper_fluid.py#L69-L80


Could you add this to the solver wrapper and call this function to keep the same structure as the fluid app?

Comment (EduardGomezEscandell) [reply applications/CompressiblePotentialFlowApplication/python_scripts/potential_flow_analysis.py]: No problem

**Review by marcnunezc — APPROVED** [review]

**Pull request merged**